test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -52,6 +52,7 @@ Describe 'Testing of script internationalization' -Tags "CI" {
 
         import-localizedData mydata -uiculture nl-NL -ea SilentlyContinue -ev ev
 
+        $ev | Should Not BeNullOrEmpty
         $ev[0].Exception.GetType() | Should Be System.Management.Automation.PSInvalidOperationException
```

Comment:
I see, if it's obvious `$result` is not null, you will go with `$result | should BeOfType`. If not, you will check `$result | Should Not BeNullOrEmpty` first.
But after checking "Not BeNullOrEmpty", why not also use `$result | should BeOfType`? I see in some places you just continue to use `$result.GetType() | Should Be`.